content/docs/composition-vs-inheritance.md

```diff
@@ -161,12 +161,13 @@ class SignUpDialog extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/gwZbYa?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/gaearon/pen/gwZbYa?editors=0010)
 
+## Այսպիսով, իսկ ի՞նչ կարող ենք ասել ժառանգականության մասին {#so-what-about-inheritance}
 ## So What About Inheritance? {#so-what-about-inheritance}
 
-At Facebook, we use React in thousands of components, and we haven't found any use cases where we would recommend creating component inheritance hierarchies.
+Facebook-ում, մենք օգտագործում ենք հազարավոր կոմպոնենտներ, և մենք չենք գտել այնպիսի դեպքեր, որտեղ խորհուրդ կտայինք ստեղծել կոմպոնենտ ժառանգականության հիերարխիայով։
```

Comment:
"Facebook-ում"֊ից հետո ստորակետ պետք չի։ Սենց հասարակ տեղի պարագաները, որ սկզբում են լինում, հաստատ պետք չի ստորակետ։

Reply:
"we use React in"  Հայերենում էս React բառը չտեսա

Reply:
"կոմպոնենտ ժառանգականության հիերարխիայով", թե՞ "կոմպոնենտի ժառանգականության հիերարխիաներ"